.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Related Issue
+
+- Closes: #[issue number] (Link the issue this pull request addresses)
+
```

Comment:
Can we also have a checkbox to indicate that the user has read and followed the style guide? Will be available once this gets merged: https://github.com/kadenzipfel/smart-contract-vulnerabilities/pull/47

Reply:
Hey kadenzipfel, that's a great idea!  Having a checkbox to confirm following the style guide will definitely help with keeping the code consistent.

Here's how we can modify the pull request template to include it:
```
### Style Guide Check
- [ ] I have read and followed the style guide.
```

 please let me know if it's fine or not

Reply:
This is good but I would name the header "Checklist" instead so we can add things later and I would link to the style guide. Can add style guide link now even though it doesn't exist yet and I will just make sure to merge the PR's in order